ansible/roles/aws-kubectl-config/tasks/packages.yml

```diff
@@ -0,0 +1,128 @@
+---
+ Used for older versions of amazon linux
```

Comment:
as we upgraded to Amazon Linux 2 this should not be necessary, right?

Reply:
Correct. This is instance specific and most likely will not be needed.